.github/workflows/ci.yml

```
@@ -25,6 +25,18 @@ jobs:
         with:
           path: ${{ steps.yarn-cache-dir.outputs.YARN_CACHE_DIR }}
           key: yarn-cache-${{ runner.os }}-${{ steps.yarn-version.outputs.YARN_VERSION }}-${{ hashFiles('yarn.lock') }}
+      - name: Determine whether the current PR is a draft
+        id: set-is-draft
+        if: github.event_name == 'pull_request' && github.event.pull_request.number
+        run: echo "IS_DRAFT=$(gh pr view --json isDraft --jq '.isDraft' ${PR_NUMBER})" >> "$GITHUB_OUTPUT"
```

To fix the problem reported by shellcheck:
```suggestion
        run: echo "IS_DRAFT=$(gh pr view --json isDraft --jq '.isDraft' "${PR_NUMBER}")" >> "$GITHUB_OUTPUT"
```

---

Ah thanks. Makes me think this should get run locally....